apis/v1alpha2/httproute_types.go

```diff
@@ -113,21 +113,26 @@ type HTTPRouteSpec struct {
 
 	// TLS defines the TLS certificate to use for Hostnames defined in this
 	// Route. This configuration only takes effect if the AllowRouteOverride
-	// field is set to true in the associated Gateway resource.
+	// field is set to true in the associated Gateway Listener.
 	//
-	// Collisions can happen if multiple HTTPRoutes define a TLS certificate
-	// for the same hostname. In such a case, conflict resolution guiding
-	// principles apply, specifically, if hostnames are same and two different
-	// certificates are specified then the certificate in the
-	// oldest resource wins.
+	// Attaching a TLS certificate to HTTPRoutes provides a way for HTTPRoute
+	// owners to effectively populate certificates on Gateway Listeners. Note
+	// that only one certificate may be attached to a Gateway Listener for a
+	// specified hostname. It is not possible to use different certificates for
```

Comment:
Only one _**secret**_ can be attached to a listener, but we don't specify what is in that secret. You could have a whole pile of certificates if you want :)